dwave/optimization/include/dwave-optimization/nodes/mathematical.hpp

```diff
@@ -151,6 +151,73 @@ using NaryMaximumNode = NaryOpNode<functional::max<double>>;
 using NaryMinimumNode = NaryOpNode<functional::min<double>>;
 using NaryMultiplyNode = NaryOpNode<std::multiplies<double>>;
 
+/// TODO: support multiple axes
+template <class BinaryOp>
+class PartialReduceNode : public ArrayOutputMixin<ArrayNode> {
```

Comment:
I haven't gone back and done it for old classes. But I'd like new ones to have the methods in alphabetical order. The old ones are driving me somewhat nuts.

Reply:
Yes. IMO once the number of methods gets high enough, the grouping becomes more confusing than helpful. Most times you're looking for a specific method and being able to find it quickly is helpful.

Reply:
roger!